### PR TITLE
Packaging for PyPi

### PR DESCRIPTION
**Pull request by sam-watttime**
This PR introduces a github actions workflow to build the python package, and add it to the pypi index (permissions already setup through a linkage with github). 

Additionally, the package has been renamed once installed `watttime_sdk -> watttime_client`

I followed this tutorial for the integration. There may be some trouble shooting to get this working, but this seems like the best practice. 

https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +65,9 @@ wt_hist.get_historical_csv(
 ```
 
 You could also combine these classes to iterate through all regions where you have access to data:
+
 ```python
+from watttime_client import WattTimeMyAccess, WattTimeHistorical
 from watttime_sdk import WattTimeMyAccess, WattTimeHistorical
```
Comment (nsteins) [inline README.md]:
```suggestion

```

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -40,3 +40,81 @@ jobs:
     - name: Test with pytest
       run: |
         pytest
+    - name: Install pypa/build
```
Comment (nsteins) [inline .github/workflows/python-app.yml:43]: Do we want to build on every push? or should we run this only on tag?

Comment (sam-watttime) [reply .github/workflows/python-app.yml:43]: good catch, only on tag. I'll make the change.

**Review by nsteins — APPROVED** [review]

File: .github/workflows/python-app.yml (hunk 2 above)
Comment (sam-watttime) [reply .github/workflows/python-app.yml:43]: actually, keeping this as is. building only takes a few seconds, and requires the same container as our testing, so it makes sense to just get it done on PR (perhaps something in the setup.py changed affecting the ability to build), publishing the build to PyPi only happens on tag/release

**Pull request merged**